test/dtest/ccmlib/common.py

```diff
@@ -0,0 +1,11 @@
+#
+# Copyright (C) 2024-present ScyllaDB
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+#
+
+import sys
+
+
+def is_win():
+    return sys.platform in ("cygwin", "win32")
```

Comment:
I can't believe you actually need this function - Scylla doesn't work on Windows so this can't be useful.
Please explain in the commit message what is the strategy you used here. Are you copying ccmlib and dtest as-is (and this is why you got this is_win() function), or rewriting it from scratch using the test/pylib "Scylla Manager"?

Maybe you can even explain in the commit message which files you copied as-is, which files are new test/pylib wrappers, or what. There's a lot of code here, I don't know how much is just copied pre-existing already-tested dtest code, and how much is new code.

Reply:
If we apply such cleanups during the migration, it will never end.

Reply:
There's a difference between a "cleaup" (which I agree we don't need to do right now) and just not copying an unused function. If this function **is** actually used in the dtest code, than my condolences and I agree we don't need to clean it up now and can leave it.

Reply:
Yep, it should be removed. On this stage of working on this PR I'm trying to minimize modifications to dtest code.